ui/src/app/views/workspace/component.ts

```diff
@@ -204,6 +195,19 @@ export class WorkspaceComponent implements OnInit, OnDestroy {
     this.bugReportOpen = true;
   }
 
+  workspaceClustersBillingProjectId(): string {
+    if (this.workspace.namespace === this.freeTierBillingProject) {
+      return this.freeTierBillingProject;
+    }
+
+    if (this.accessLevel === WorkspaceAccessLevel.WRITER ||
```

Comment:
opt_nit: `[WorkspaceAccessLevel.WRITER, WorkspaceAccessLevel.OWNER].includes(this.accessLevel)`